h/security/permits/demo.py

```diff
@@ -0,0 +1,52 @@
+if __name__ == "__main__":
```

Comment:
Some random demo code you can run to exercise the system. I was using this for benchmarking